airflow/utils/dag_processing.py

```diff
@@ -308,6 +369,249 @@ def file_path(self):
         raise NotImplementedError()
 
 
+class DagParsingStat(object):
```

Comment:
same, why not namedTuple?

Reply:
This is definitely a better candidate for namedTuple, will try update. Ty.